openapi/openapi-client/src/main/java/org/apache/celeborn/rest/v1/master/ApplicationApi.java

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
Should we include the Apache License header in the auto-generated Java files?

Reply:
I tried, but the tool does not support it.

Reply:
FYI:
https://github.com/OpenAPITools/openapi-generator/issues/7535#issuecomment-712942919

Reply:
after look into the maven settings and openapi-generator code, added the license.